modules/hivemq-edge-module-file/src/main/java/com/hivemq/edge/adapters/file/FilePollingProtocolAdapter.java

```diff
@@ -88,7 +93,27 @@ public void stop(
     @Override
     public void poll(
             final @NotNull PollingInput<FileToMqttMapping> pollingInput, final @NotNull PollingOutput pollingOutput) {
-        final String absolutePathToFle = pollingInput.getPollingContext().getFilePath();
+        final ProtocolAdapterTagService protocolAdapterTagService = pollingInput.protocolAdapterTagService();
+
+        final String tagName = pollingInput.getPollingContext().getTagName();
+        final Tag<FileTagDefinition> fileTag;
+        try {
+            fileTag= protocolAdapterTagService.resolveTag(pollingInput.getPollingContext().getTagName(),
+                    FileTagDefinition.class);
+        } catch (final TagNotFoundException e) {
```

Comment:
As mentioned above, I don't think an exception is appropriate way to handle this.
We should probably create something to make this also visible in the UI.